packages/components/components/elvis-dropdown/src/react/elvia-dropdown.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import percySnapshot from '@percy/playwright';
+import { expect, test } from '@playwright/experimental-ct-react';
+import React from 'react';
+
+import { Dropdown } from '../../react';
+
+test.use({ viewport: { width: 1080, height: 300 } });
+
+const items = [
+  { value: 'Norge', label: 'Norge' },
+  { value: 'Sverige', label: 'Sverige' },
+  { value: 'Danmark', label: 'Danmark' },
+];
+
+const nestedItems = [
+  {
+    value: 'norge',
+    label: 'Norge',
+    children: [
+      {
+        label: 'Bergen',
+        value: 'bergen',
+        children: [
+          { label: 'Arna', value: 'arna' },
+          { label: 'Bergenhus', value: 'bergenhus' },
+        ],
+      },
+      { label: 'Trondheim', value: 'trondheim' },
+    ],
+  },
+  { value: 'spania', label: 'Spania' },
+  { value: 'tyskland', label: 'Tyskland' },
+];
+
+test('should render', async ({ mount }) => {
+  const component = await mount(<Dropdown value={'Norge'} label={'Land'} items={items} />);
+  await expect(component).toBeAttached();
+});
+
+test('should look closed', async ({ mount, page }) => {
+  await mount(<Dropdown value={'Norge'} label={'Land'} items={items} />);
+  await percySnapshot(page, 'Closed Dropdown');
+});
+
+test('should look opened', async ({ mount, page }) => {
+  const component = await mount(<Dropdown value={'Norge'} label={'Land'} items={items} />);
+
+  await component.getByRole('combobox').click();
+  await percySnapshot(page, 'Open Dropdown');
+});
```

Comment:
Each call to `percySnapshot` will lead to a new (unique) snapshot. Because of the billing model, it may be smart to avoid making too many small and simple snapshot tests. I would therefore consider combining these two into one test.
For larger tests where theres more interaction I would still argue that it makes sense to only have one component per test 😄